examples/public/dynamic-share.php

```diff
@@ -6,10 +6,28 @@
 
 use Yoti\ShareUrl\DynamicScenarioBuilder;
 use Yoti\ShareUrl\Extension\LocationConstraintExtensionBuilder;
+use Yoti\ShareUrl\Policy\ConstraintsBuilder;
 use Yoti\ShareUrl\Policy\DynamicPolicyBuilder;
+use Yoti\ShareUrl\Policy\SourceConstraintBuilder;
 
 $yotiClient = new Yoti\YotiClient(YOTI_SDK_ID, YOTI_KEY_FILE_PATH);
 
+$drivingLicenseConstraint = (new ConstraintsBuilder())
+    ->withSourceConstraint(
+        (new SourceConstraintBuilder())
+            ->withDrivingLicence()
+            ->build()
+    )
+    ->build();
+
+$passportConstraint = (new ConstraintsBuilder())
+    ->withSourceConstraint(
+        (new SourceConstraintBuilder())
+            ->withPassport()
+            ->build()
+    )
+    ->build();
```

Comment:
I think we decided we wouldn't, as it will only be a small subset of our client base who will want this, and it will put further restrictions on who can run the Dynamic Share demo

Reply:
Should we remove the source constraints example completely? If we don't want to demonstrate multiple attributes with different constraints, we could instead just have a single example of retrieving document details from a passport to simplify? I can also remove document images from the policy.

Reply:
Removed constraints in f39b11f